### PR TITLE
feat(ci.jenkins.io): add a window agent ACI with jdk17

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/2822
create a windows jdk 17 agent based on the docker image : https://hub.docker.com/layers/jenkinsciinfra/inbound-agent-maven/jdk17-nanoserver/images/sha256-53ec72a0a97adfb3b8f901d49f82c419cf9177516a2f488612135c186ea27170?context=explore

### CODE REVIEW

**Comment by smerle33** [comment]
do not merge after approval, I want to check on puppet on the ci VM

**Review by dduportal — APPROVED** [review]
🚀

**Pull request merged**
